### PR TITLE
Add sBTC shorts, add inactive | active category for incentives

### PR DESCRIPTION
**Pull request by andytcf**
image

image

image

image

### CODE REVIEW

File: hooks/useShortRewardsData.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import useExchangeRatesQuery from 'queries/rates/useExchangeRatesQuery';
+import { ShortRewardsData } from 'queries/shorts/types';
+import useSBTCShortsQuery from 'queries/shorts/useSBTCShortsQuery';
+
+type SRData = {
+	[name: string]: {
+		APR: number;
+		OI: number;
+		data: ShortRewardsData | undefined;
+	};
```
Comment (evgenyboxer) [inline hooks/useShortRewardsData.ts:12]: Can also be a `Record<string, { .. }` - but both are cool :)

**Review by evgenyboxer — APPROVED** [review]
@andytcf lgtm!

**Review by JChiaramonte7 — APPROVED** [review]

**Review by hav-noms — APPROVED** [review]

**Review by evgenyboxer — APPROVED** [review]
lgtm

**Comment by andytcf** [comment]
@andytcf change tab language from "active" | "expired"

**Pull request merged**